### PR TITLE
fix: third-party test

### PR DESCRIPTION
**Pull request by jiashenC**
1. Added sqlite testcases to PR testing.
2. Fixed `mysql` case sensitivity issue. 
3. Fixed the `sqlalchemy` uri for each database.

### CODE REVIEW

**Comment by jiashenC** [comment]
Temporarily enable third party tests to ensure CI passes. But, I think we should maybe enable one `Postgres` even for open PR to prevent future database integration issues being merged to `staging`?

**Comment by jiashenC** [comment]
https://github.com/georgia-tech-db/evadb/issues/1097

https://github.com/georgia-tech-db/evadb/issues/1117

**Review by xzdandy — APPROVED** [review]

File: test/third_party_tests/test_native_executor.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def _create_table_in_native_database(self):
             """USE test_data_source {
                 CREATE TABLE test_table (
                     name VARCHAR(10),
-                    Age INT,
```
Comment (gaurav274) [inline test/third_party_tests/test_native_executor.py:41]: I intentionally made this change so that we could test the corner case.

File: .circleci/config.yml [hunk 2]
```diff
@@ -94,22 +94,22 @@ workflows:
         ################################
         - Postgres:
             name: Third Party Test - Postgres | v3.10 | Linux
-            filters:
-              branches:
-                only: 
-                  - master
-                  - staging
+            # filters:
```
Comment (gaurav274) [inline .circleci/config.yml:97]: Why are we commenting this out?

**Comment by gaurav274** [comment]
> Temporarily enable third party tests to ensure CI passes. But, I think we should maybe enable one `Postgres` even for open PR to prevent future database integration issues being merged to `staging`?

How about `sqlite`? It does not need any setup overhead and will likely cover most cases.

File: test/third_party_tests/test_native_executor.py (hunk 1 above)
Comment (xzdandy) [reply test/third_party_tests/test_native_executor.py:41]: Ah I missed that. Thanks for point it out.

File: script/test/test.sh [hunk 3]
```diff
@@ -88,13 +94,14 @@ notebook_test() {
...
-  PYTHONPATH=./ pytest test/ --durations=20 --cov-report term-missing:skip-covered  --cov-config=.coveragerc --cov-context=test --cov=evadb/ --capture=sys --tb=short -v -rsf --log-level=WARNING -m "not benchmark" --ignore=test/third_party_tests/ --ignore=test/app_tests/
+  PYTHONPATH=./ pytest test/ test/third_party_tests/test_native_executor.py::NativeExecutorTest::test_should_run_query_in_sqlite --durations=20 --cov-report term-missing:skip-covered  --cov-config=.coveragerc --cov-context=test --cov=evadb/ --capture=sys --tb=short -v -rsf --log-level=WARNING -m "not benchmark" --ignore=test/third_party_tests/ --ignore=test/app_tests/
   code=$?
+  
   print_error_code $code "FULL TEST"
 }
 
 no_coverage_full_test() {
-  PYTHONPATH=./ python -m pytest -p no:cov test/ -m "not benchmark" --ignore=test/third_party_tests/ --ignore=test/app_tests/
+  PYTHONPATH=./ python -m pytest -p no:cov test/ test/third_party_tests/test_native_executor.py::NativeExecutorTest::test_should_run_query_in_sqlite-m "not benchmark" --ignore=test/third_party_tests/ --ignore=test/app_tests/
```
Comment (jiashenC) [inline script/test/test.sh:104]: Are we missing a space here? ` ... _in_sqlite-m`

**Comment by jiashenC** [comment]
Also is MySQL CI running fine now?
